src/Form.php

```diff
@@ -1665,21 +1720,29 @@ protected function loadField($element, $group = null, $value = null)
 		{
 			$default = (string) $element['default'];
 
-			if (($translate = $element['translate_default']) && ((string) $translate == 'true' || (string) $translate == '1'))
+			// Try to translate the default value if translations are enabled
+			try
 			{
-				$lang = Language::getInstance();
+				$lang = $this->getText()->getLanguage();
 
-				if ($lang->hasKey($default))
-				{
-					$debug = $lang->setDebug(false);
-					$default = Text::_($default);
-					$lang->setDebug($debug);
-				}
-				else
+				if (($translate = $element['translate_default']) && ((string) $translate == 'true' || (string) $translate == '1'))
 				{
-					$default = Text::_($default);
+					if ($lang->hasKey($default))
+					{
+						$debug = $lang->setDebug(false);
```

Comment:
Is there any reason for turning debug off and back on here. I know that's how we did it before. But it doesn't really make sense...